src/ape/contracts/base.py

```diff
@@ -628,6 +630,28 @@ def poll_logs(
             yield from self.range(new_block.number, stop=new_block.number + 1)
 
 
+class ContractError(ManagerAccessMixin):
```

Comment:
Hmm, maybe can call this `CustomError` or `ContractErrorType` to avoid having to alias the `ContractError` exception as an import

Reply:
Also, don't think it needs `ManagerAccessMixin`